tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/stack_trace/stack_trace_component.css

```diff
@@ -45,6 +45,8 @@ limitations under the License.
   height: 300px;
   overflow-x: hidden;
   overflow-y: auto;
+  /* Minus padding-left of the parent (stack-trace-container). */
+  width: calc(100% - 8px);
```

Comment:
instead of keeping track of all the padding of parents in mind, just use flex (with justify-content: stretch) where possible.

Reply:
Thanks for the suggestion. Normally yes, but StackTraceComponent is already flex in the column (vertical direction), which makes horizontal stretching a little complex. It's less complex to do it this way.